packages/backend.ai-ui/src/components/baiClient/FileExplorer/DeleteSelectedItemsModal.tsx

```diff
@@ -49,11 +55,19 @@ const DeleteSelectedItemsModal: React.FC<DeleteSelectedItemsModalProps> = ({
         recursive: true,
         name: targetVFolderId,
       })
-      .then(() => {
-        onRequestClose(true);
-        message.success(
-          t('comp:FileExplorer.SelectedItemsDeletedSuccessfully'),
-        );
+      .then(({ bgtask_id }: { bgtask_id: string }) => {
+        onRequestClose(true, selectedFileNames);
+        if (bgtask_id) {
+          onDeleteFilesInBackground?.(
+            bgtask_id,
+            targetVFolderId,
+            selectedFileNames,
```

Comment:
`selectedFileNames`​ include file names but, `deletingFilePaths`​ need whole path. Is it right?

Reply:
https://github.com/lablup/backend.ai-webui/blob/c322e28fd193308b853cf7843a6c563b327a3572/packages/backend.ai-ui/src/components/baiClient/FileExplorer/DeleteSelectedItemsModal.tsx#L49-L51  
Please check this code. It includes path.